cli/tests/unit/streams_test.ts

```diff
@@ -249,9 +247,9 @@ Deno.test(async function readableStreamLongAsyncReadAll() {
 });
 
 Deno.test(async function readableStreamVeryLongReadAll() {
-  const rid = resourceForReadableStream(veryLongTinyPacketStream(10000));
+  const rid = resourceForReadableStream(veryLongTinyPacketStream(1_000_000));
```

Comment:
@mmastrac let me know if you prefer a lower number.

Reply:
I think 1000000 is good even if it's a few seconds to test. It really exercises the large queues.